recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     - py-cpuinfo
     - python >=3.7
     - rich >=11.0.0
-    - textual >=0.1.15,<0.2
+    - textual >=0.1.14,<0.2
```

Comment:
According to pyproject.toml it should be 0.1.15. My local grayskull run should that:

```
dependencies = [
   "py-cpuinfo",
   "distro",
   "psutil",
   "rich >= 11.0.0",
   "textual >=0.1.15, <0.2",
]
```

Reply:
Not a big deal. We can do that in the next release.

Reply:
I bumped the build anyway, it's merged.